docker/geneformer/model.py

```diff
@@ -217,8 +217,8 @@ def _extract_embeddings(self, tokenized_dataset_path: Path, dataset: BaseDataset
         embex = EmbExtractor(
             model_type="Pretrained",
             emb_layer=-1,
-            emb_mode="cell",
-            forward_batch_size=32,
+            emb_mode="cls",
+            forward_batch_size=16,
```

Comment:
I reduced the forward batch size since with the corrected weights, it takes much more memory